lib/core/app/operations/ros/cloud_event_stream.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Ros
+  class CloudEventStream < Ros::ActivityBase
+    step :log_event
+    # fail :log_error
+
+    def log_event(_ctx, type:, message_id:, data:, **)
+      Rails.configuration.x.event_logger.log_event(type, message_id, data) # => Faraday::Response
+      true
```

Comment:
@rpbaltazar do we need `true` here if `Faraday::Response` is truthy itself?

Reply:
No. As long as the returning value of the log_event is truthy/falsy then the outcome of the operation should be success or failure as well. 
In case log_event returns false, are there any errors in that object that we can log or we'll have to write our own error message?